using-ide.md

```diff
@@ -230,10 +230,10 @@ You can also define custom templates in **Settings > Editor > Live Templates**.
 **Hot Reload**
 
 On Linux (keymap _Default for XWin_) and Windows the keyboard shortcuts
-are `ctrl-alt-;` and `ctrl-\`.
+are `ctrl`+`alt`+`;` and `ctrl`+`\`.
 
-On macOS (keymap _Mac OS X 10.5+ copy_) the keyboard shortcuts are `⌘-⌥-;`
-and `⌘-\`.
+On macOS (keymap _Mac OS X 10.5+ copy_) the keyboard shortcuts are `⌘`+`⌥`+`;`
```

Comment:
We don't use keyboard shortcuts in our docs. Please delete.

Reply:
Should I delete all the keyboard shortcuts?